src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/MultiBindingCondition.shared.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Xamarin.CommunityToolkit.Helpers
+{
+	public enum MultiBindingCondition
+	{
+		All = 0,
+		Any = 1,
+		Count = 2
+	}
```

Comment:
```csharp
	[Flags]
	public enum MultiBindingCondition
	{
		None = 1,
		Any = 2,
		All = 4,
		Count = 8
	}
```

In this way, I can get "True" if **None** of converters returned **True** (All false)

Or, for example, **None** or **All** True, otherwise False (Flags combination)